src/applications/check-in/tests/e2e/support/commands.js

```diff
@@ -0,0 +1,74 @@
+import mockCheckIn from '../../../api/local-mock-api/mocks/v2/check.in.responses';
```

Comment:
@mdewey just updated the commands file. I didn't want to have a bunch of if statements or switches and this will let us use both at the test level if we need to sunset versions slower. Let me know what you think.